notebooks/COS/DayNight/DayNight.ipynb

```diff
@@ -208,32 +313,46 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "execution": {
```

Comment:
I noticed a lot of these `"exection"` entries here.  I'm not sure what that means, but is it possible you didn't clear outputs or something on one of these notebooks? (if you're not sure either, it's probably not an issue - just thought I'd check)

Reply:
Ah, I believe this is from the way that I cleared my cells - I wrote a script that clears Notebooks with `jupyter-nbconvert --ClearOutputPreprocessor.enabled=True --inplace <Notebook.ipynb>` which I believe leaves them like that.